src/base/ApplicationVersionedLambda.ts

```diff
@@ -193,7 +193,7 @@ export class ApplicationVersionedLambda extends Resource {
     let filename = `${functionFilename}.js`;
 
     if (runtime === 'python') {
-      content = `${functionName}(event, context):\n\t print(event)`;
+      content = `import json\ndef ${functionName}(event, context):\n\t print(event)\n\t return {'statusCode': 200, 'headers': {'dance': 'party'}, 'body': json.dumps({'electric': 'boogaloo'}), 'isBase64Encoded': False}`;
```

Comment:
why does python runtime have this special requirement?

Reply:
can you clarify what you mean by special requirement?

Reply:
sure, i am understanding this code as we check if the code running in lambda is python, and if yes, then we are returning an object with status code and specific header. `return {'statusCode': 200, 'headers': {'dance': 'party'}, 'body': json.dumps({'electric': 'boogaloo'}), 'isBase64Encoded': False}`;`why are we doing this?

Reply:
Oh! I get you. This just generates some code for a default lambda handler. In reality you would provide your own function.

I only had to edit this because originally it was syntactically incorrect python, and it didn't return a response in the expected format (causing an error when testing).